Assets/_Scripts/Character/StatusAilment/StatusAilment.cs

```diff
@@ -4,11 +4,24 @@
 
 public abstract class StatusAilment : ScriptableObject {
 	public string name;
-	public int duration;
+	[Tooltip("Duration used if the action inflicting this status ailment doesn't specify duration. Can leave this at 0.")]
+	public int defaultDuration;
+	[HideInInspector] public int duration;
 	public int level;
-	[Header("Phase to trigger Status Ailment")]
+	[Tooltip("Phase to trigger Status Ailment")]
 	public BattlePhase phase;
 
+	private void Awake() {
+		// Set duration to default if one is not set
+		if (duration == 0 && defaultDuration != 0) {
+			duration = defaultDuration;
+		}
+	}
+
+	public void SetDuration(int duration) {
```

Comment:
Can we get a specific option to set it to infinite duration?

Reply:
Good idea, added